src/main/java/mServer/crawler/sender/br/BrUrlDTO.java

```diff
@@ -0,0 +1,70 @@
+package mServer.crawler.sender.br;
+
+public class BrUrlDTO {
+  private final String url;
+  private final int width;
+  private final String videoProfile;
+
+  public BrUrlDTO(final String aUrl, final int aWidth, final String aVideoProfile) {
+    super();
+    url = aUrl;
+    width = aWidth;
+    videoProfile = aVideoProfile;
+  }
+
+  @Override
+  public boolean equals(final Object obj) {
+    if (this == obj) {
+      return true;
+    }
+    if (obj == null) {
+      return false;
+    }
+    if (getClass() != obj.getClass()) {
+      return false;
+    }
+    final BrUrlDTO other = (BrUrlDTO) obj;
+    if (url == null) {
+      if (other.url != null) {
+        return false;
+      }
+    } else if (!url.equals(other.url)) {
+      return false;
+    }
+    if (videoProfile == null) {
+      if (other.videoProfile != null) {
+        return false;
+      }
+    } else if (!videoProfile.equals(other.videoProfile)) {
+      return false;
+    }
+    if (width != other.width) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this if-then-else statement by a single return statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1126)